### PR TITLE
Add listTask function and fix UI bug to display storage of task in task tab

### LINKED ISSUE #48: Show outstanding tasks in a room: listTask
As a forgetful staff, I can look at the remaining tasks my co-workers and I have to complete to complete for a particular hotel room.

### PR DESCRIPTION
**Pull request by raymondge**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -15,24 +15,24 @@ title: User Guide
...
+          5.3.3  [Search by Room Number: `searchroom`](#533-search-by-room-number-searchroom)<br>
+          5.3.4  [Search for Room with Patient: `searchroom`](#534-search-for-room-with-patient-searchroom)<br>
+          5.3.5  [Lists the current rooms: `listRoom`](#535-lists-the-current-rooms-listroom)<br>
+          5.3.6  [Find the first free room: `findRoom`](#536-find-the-first-free-room-findroom)<br>
     5.4 [Task](#54-task)<br>
-        5.4.1  [Add a task to a room: `addtask`](#541-add-a-task-to-a-room-addtask)<br>
-        5.4.2  [Delete a task from a room: `deletetask`](#542-delete-a-task-from-a-room-deletetask)<br>
-        5.4.3  [Edit a task in a room: `edittask`](#543-edit-a-task-in-a-room-edittask)<br>
-        5.4.4  [Searches all tasks before the given date: `searchtask`](#544-search-tasks-before-a-date-searchtask)<br>
+          5.4.1  [Add a task to a room: `addtask`](#541-add-a-task-to-a-room-addtask)<br>
```
Comment (LeeMingDe) [inline docs/UserGuide.md:32]: thanks for indenting!

File: src/main/java/seedu/address/model/ModelManager.java [hunk 2]
```diff
@@ -328,12 +328,20 @@ public void setTask(Task taskToEdit, Task editedTask) {
     public void updateFilteredTaskList(Predicate<Task> predicate) {
         requireNonNull(predicate);
         filteredTasks.setPredicate(predicate);
+        for (Task task : filteredTasks) {
+            System.out.println("1" + task);
```
Comment (LeeMingDe) [inline src/main/java/seedu/address/model/ModelManager.java]: is this system println needed?

**Review by LeeMingDe — APPROVED** [review]
lgtm! maybe you want to remove the `System.out.println`?

File: src/main/java/seedu/address/model/ModelManager.java (hunk 2 above)
Comment (raymondge) [reply src/main/java/seedu/address/model/ModelManager.java]: thx for noticing! I forgot to remove

**Pull request merged**